### PR TITLE
Build the library with Travis CI

### PR DESCRIPTION
**Pull request by hectorerb**
This PR generate the build with Travis CI

Ref: https://github.com/flyve-mdm/flyve-mdm-ios-inventory/issues/4

### CODE REVIEW

File: FlyveMDMInventoryTests/FlyveMDMInventoryTests.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Hector Rondon
+ * @date      06/06/17
+ * @copyright Copyright © 2017 Teclib. All rights reserved.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @link      https://github.com/flyve-mdm/flyve-mdm-ios-inventory
+ * @link      http://www.glpi-project.org/
```
Comment (ajsb85) [inline FlyveMDMInventoryTests/FlyveMDMInventoryTests.swift]: https://flyve-mdm.com

File: FlyveMDMInventory/FlyveMDMInventory.h [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Hector Rondon
+ * @date      06/06/17
+ * @copyright Copyright © 2017 Teclib. All rights reserved.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @link      https://github.com/flyve-mdm/flyve-mdm-ios-inventory
+ * @link      http://www.glpi-project.org/
```
Comment (ajsb85) [inline FlyveMDMInventory/FlyveMDMInventory.h]: https://flyve-mdm.com

File: FlyveMDMInventory.podspec [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  #  accepts just a name if you'd rather not provide an email address.
+  #
+  #  Specify a social_media_url where others can refer to, for example a twitter
+  #  profile URL.
+  #
+
+  s.author             = { "Flyve MDM" => "apple@teclib.com" }
+  # Or just: s.author    = "Hector Rondon"
+  # s.authors            = { "Hector Rondon" => "hectorerb@gmail.com" }
+  # s.social_media_url   = "https://twitter.com/FlyveMDM"
```
Comment (ajsb85) [inline FlyveMDMInventory.podspec]: https://twitter.com/FlyveMDM

File: FlyveMDMInventory.podspec [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Licensing your code is important. See http://choosealicense.com for more info.
+  #  CocoaPods will detect a license file if there is a named LICENSE*
+  #  Popular ones are 'MIT', 'BSD' and 'Apache License, Version 2.0'.
+  #
+
+  # s.license      = "GPLv3"
+  s.license      = { :type => "GPLv3", :file => "LICENSE" }
```
Comment (ajsb85) [inline FlyveMDMInventory.podspec]: The file is: `LICENSE.md`

File: FlyveMDMInventoryTests/FlyveMDMInventoryTests.swift (hunk 1 above)
Comment (hectorerb) [inline FlyveMDMInventoryTests/FlyveMDMInventoryTests.swift]: Updated!

File: FlyveMDMInventory/FlyveMDMInventory.h (hunk 2 above)
Comment (hectorerb) [inline FlyveMDMInventory/FlyveMDMInventory.h]: Updated!

File: FlyveMDMInventory.podspec (hunk 3 above)
Comment (hectorerb) [inline FlyveMDMInventory.podspec]: Updated!

File: FlyveMDMInventory.podspec (hunk 4 above)
Comment (hectorerb) [inline FlyveMDMInventory.podspec]: Updated!

**Pull request merged**